src/pages/dho/Organizational.vue

```diff
@@ -91,6 +94,60 @@ export default {
         }
       }
     },
+    daoPayouts: {
+      query: require('~/query/payouts/dao-payouts.gql'),
+      update: data => {
+        return data.getDao.payout.map(payout => {
+          return {
+            title: payout.details_title_s,
+            description: payout.details_description_s,
+            docId: payout.docId,
+            payments: payout.payment
+          }
+        })
+      },
+      variables () {
+        return {
+          daoId: this.selectedDao.docId,
+          first: 3
+        }
+      }
+    },
+    daoBadgeAssignments: {
+      query: require('~/query/assignments/dao-badge-assignments.gql'),
+      update: data => {
+        return data.queryAssignbadge.map(assignment => {
+          return {
+            title: assignment.details_title_s,
+            description: assignment.details_description_s,
+            docId: assignment.docId,
+            icon: assignment.badge[0].details_icon_s
+          }
+        })
+      },
+      variables () {
+        return {
+          daoName: this.selectedDao.name
+        }
+      }
+    },
+    daoRoleAssignments: {
+      query: require('~/query/assignments/dao-role-assignments.gql'),
+      update: data => {
+        return data.queryAssignment.map(assignment => {
+          return {
+            title: assignment.details_title_s,
+            description: assignment.details_description_s,
+            docId: assignment.docId
+          }
+        })
+      },
+      variables () {
+        return {
+          daoName: this.selectedDao.name
```

Comment:
We should pass params to get just the last 3 items for each proposal type

Reply:
I'm going to add the variable